### ISSUE TITLE
Flog v2 Issues - README

### ISSUE THREAD

**Issue by rbong** [rbong/vim-flog#91]
Flog v2 has been released and it comes with lots of changes.

### How does it look?

![the v2 commit graph](https://raw.githubusercontent.com/rbong/vim-flog/e091716195df49dd925120b5f7fd2fccc974f6b6/img/screen-graph.png)

### How can I upgrade to v2?

See the [v2 migration guide](https://github.com/rbong/vim-flog/wiki/v2-Migration-Guide)

You will also receive deprecation warnings when possible

### How can I revert back to v1?

Install Flog using the branch `v1`, ex. `Plug 'rbong/vim-flog', { 'branch': 'v1' }`

### I used the alpha, how can I keep using v2?

The `v2` branch that was being used during alpha is no longer available, switch back to the master (default) branch.
In other words, install normally, ex. `Plug 'rbong/vim-flog'`

### Why did everything change?

Flog was essentially a "hack" before, a wrapper around the system command `git log --graph`, making it slow, ugly, and limited.
Fixing this requires drawing our own commit graph, which requires a real language, ex. Lua.
Adding a new major dependency is not something everyone will like, which is why `v1` is still available

Flog also had a number of other longtime issues that required breaking changes, which are being released at the same time

### Other

Please post a [new issue](https://github.com/rbong/vim-flog/issues/new/choose) for any bugs, feedback, or questions
